bin/cron/update_hoc_map

```diff
@@ -37,7 +37,14 @@ begin
       next if data['organization_name_s'].nil? || processed_data['location_p'].nil?
       next if form[:review] != "approved" && processed_data['location_city_s'].nil?
 
-      row = [form[:id], data['organization_name_s'], processed_data['location_city_s'], processed_data['location_p'], form[:review], data['special_event_details_s']]
+      if !processed_data['nces_school_name_s'].to_s.strip.empty?
+        organization_name = processed_data['nces_school_name_s']
+      elsif !data['school_name_s'].to_s.strip.empty?
+        organization_name = data['school_name_s']
+      else
+        organization_name = data['organization_name_s']
+      end
```

Comment:
I've been told to be `!` averse in ruby, could do something like:
```
organization_name ||= processed_data['nces_school_name_s'].to_s.strip.presence
organization_name ||= data['school_name_s'].to_s.strip.presence
organization_name ||= data['organization_name_s'].to_s.strip.presence
```

Reply:
This isn't Rails so I can't use presence. I haven't found a pure Ruby way to evaluate not_empty without ````!````